src/components/CompareResults/SubtestsResults/SubtestsRevisionHeader.tsx

```diff
@@ -0,0 +1,143 @@
+import { type ReactNode } from 'react';
+
+import AppleIcon from '@mui/icons-material/Apple';
+import { Link } from '@mui/material';
+import { style } from 'typestyle';
+
+import { Strings } from '../../../resources/Strings';
+import { Colors, Spacing } from '../../../styles';
+import type {
+  SubtestsRevisionsHeader,
+  PlatformShortName,
+} from '../../../types/state';
+import {
+  getTreeherderURL,
+  truncateHash,
+  getDocsURL,
+} from '../../../utils/helpers';
+import { getPlatformShortName } from '../../../utils/platform';
+import AndroidIcon from '../../Shared/Icons/AndroidIcon';
+import LinuxIcon from '../../Shared/Icons/LinuxIcon';
+import WindowsIcon from '../../Shared/Icons/WindowsIcon';
+
+const styles = {
+  revisionHeader: style({
+    display: 'flex',
+    alignItems: 'center',
+    paddingBottom: '12px',
+    marginBottom: '12px',
+  }),
+  tagsOptions: style({
+    textAlign: 'right',
+    $nest: {
+      'span:nth-child(3n)': {
+        background: '#592ACB',
+      },
+      'span:nth-child(3n+1)': {
+        background: '#005E5E',
+      },
+      'span:nth-child(3n+2)': {
+        background: '#0250BB',
+      },
+    },
+  }),
+  chip: style({
+    borderRadius: '4px',
+    color: Colors.Background300,
+    // To be removed
+    fontFamily: 'SF Pro',
+    fontStyle: 'normal',
+    fontWeight: 700,
+    fontSize: '8.2px',
+    // End to be removed
```

Comment:
to be removed?

Reply:
This is something I left the same as in the RevisionHeader component